backend/sozisel/test/sozisel_web/schema/session_templates/session_template_queries_test.exs

```diff
@@ -0,0 +1,115 @@
+defmodule SoziselWeb.Schema.SessionTemplateQueryTest do
+  use SoziselWeb.AbsintheCase
+
+  import Sozisel.Factory
+
+  @basic_search_template """
+  query SearchTemplate {
+    searchSessionTemplates {
+      id
+      name
+      estimatedTime
+      isAbstract
+      isPublic
+      owner {
+        id
+      }
+      agendaEntries {
+        id
+        name
+        startMinute
+      }
+    }
+  }
+  """
+
+  @search_template_with_params """
+  query SearchPublicTemplates($include_public: Boolean!, $name: String!) {
+    searchSessionTemplates(include_public: $include_public, name: $name) {
```

Comment:
Absinthe never generates anything in snake case, maybe enums but I can't see them here. So to speak, should be `includePublic`. Remember that anything facing frontend should be in camel case and that is what absinthe does, it changes all elixir's snake cases to matching camel cases.

Reply:
And again, here you refer to searching by name (which is correct) but when it comes to functions touching database you use `pattern` (incorrect). Be consistent with what you do.